cider/exec/nextgen/jitlib/llvmjit/LLVMJITUtils.h

```diff
@@ -50,6 +50,27 @@ inline llvm::Type* getLLVMType(JITTypeTag tag, llvm::LLVMContext& ctx) {
   }
 }
 
+inline llvm::PointerType* getLLVMPtrType(JITTypeTag sub_tag, llvm::LLVMContext& ctx) {
```

Comment:
seems we don't support pointer of pointer?

Reply:
It could be represented as POINTER type with POINTER sub_type.

Reply:
then how do we get sub_type of the sub_type, do we need this operation?

Reply:
I think it is unnecessary, the caller must know the actual type of the pointer.